scanpy_recipes/templates/cellview_how_to.txt

```diff
@@ -0,0 +1,12 @@
+Visualize gene expression per cluster
+    You can visualize the gene expression in a chosen cluster for one or multiple genes, under the "Explore" tab.
+        There are 2 sections under the Explore tab:
+            -Expression (1 gene | 1 cluster)
+                Use Expession to visualize the expression of one gene in one cluster at a time.
+                -Enter a gene of interest and select a cluster from the drop-down list. click "Run" to display a 3 panel visual output.
+                    -Panel 1 has a 3D plot of the gene’s expression across all cells in the cluster of interest.
+                    -Panel 2 has a 2D plot of the same gene’s expression across all cells in that cluster. Within Panel 2, you can also download a .csv file of the gene-cell expression matrix by clicking and dragging over a group of cells. click "Download Expression" at the top of panel 2 to start the download.
+                    -Panel 3 has violin plots of the gene’s cluster-specific expression and includes a total cell count for each cluster (bottom of violin plots) as well as the count of cells expressing the gene in that cluster (top of violin plots).
+            -Panel Plot (multiple genes |  1 cluster)
+                Use Panel Plot to visualize the expression of multiple genes of interest in one cluster at a time.
+                -After selecting a cluster from the drop-down list, paste a coma separated list of gene names, click "Run". 
```

Comment:
For this, could you use [markdown](https://daringfireball.net/projects/markdown/basics) or some basic html formatting just so I get the sense for how you would like this formatted?  Ultimately these will sit in a some bootstrap-styled html like:
```{html}
<div class="row">
    <div class="col">
        <div class="card-body">
            <h5 class="card-title">Visualize gene expression per cluster</h5>
            <p class="card-text">
                You can visualize the gene expression in a chosen cluster for one or multiple genes...
            </p>
       </div>
</div>
```
so you can wrap it in that if you like.